package.json

```diff
@@ -9,31 +9,31 @@
   },
   "scripts": {
     "build:dev": "webpack --mode development --watch",
-    "build:prod": "webpack --mode production --watch",
+    "build:prod": "webpack --mode production",
     "start:dev": "nodemon server/index.js",
+    "start": "node server/index.js",
     "seed:db": "node database/seed.js",
     "reset:db": "mysql -u root < database/schema.sql",
     "test": "jest --detectOpenHandles"
   },
   "dependencies": {
     "jquery": "^3.4.1",
+    "json2csv": "^4.5.2",
+    "knex": "^0.19.1",
+    "pg": "^7.11.0",
+    "pg-copy-streams": "^2.2.2",
```

Comment:
Do you use this anywhere? I didn't see it being required in any of your changes

Reply:
Good catch.  I couldn't get the csv read to work so ended up going another route.  Will remove this one